### DISCUSSION TITLE
[@types/three] ShapeUtils is defined as namespace so it can be inherit

### DISCUSSION THREAD

**Discussion by rubeniskov** [DefinitelyTyped/DefinitelyTyped#69888] [Issues with a @types package]
Instead of a class with static methods the ShapeUtils typings are defined as namespace

<img width="633" alt="Screenshot 2024-06-24 at 00 35 40" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/6261914/2ad7854d-61fc-446a-a5b2-73d6162640eb">
<img width="631" alt="Screenshot 2024-06-24 at 00 35 32" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/6261914/304b06d2-c06c-4c55-8567-869d3f6e26d4">

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment]
Thanks for creating an issue, this should be fixed by https://github.com/three-types/three-ts-types/pull/1051 and released with `@types/three@0.166.0` at the end of the month.
